code/java/index1.java

```diff
@@ -0,0 +1,103 @@
+package code.java;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.xapian.Document;
+import org.xapian.Stem;
+import org.xapian.TermGenerator;
+import org.xapian.WritableDatabase;
+import org.xapian.XapianConstants;
+import org.xapian.XapianJNI;
+
+public class index1 {
+    
+    // Command line args - dbpath datapath
+    public static void main(String[] args)
+    {
+        if(args.length < 2)
```

Comment:
I apologize for the formatting. I started getting a hang of proper formatting but I wrote this code almost a week ago and I forgot to edit everything. I'll go through everything and make sure all the formatting and alignment is proper.